packages/dolog/dolog.4.0.0/opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "2.0"
+name: "dolog"
+authors: "Francois Berenger"
+maintainer: "unixjunkie@sdf.org"
+license: "BSD-3-Clause"
+homepage: "https://github.com/UnixJunkie/dolog"
+bug-reports: "https://github.com/UnixJunkie/dolog/issues"
+dev-repo: "git+https://github.com/UnixJunkie/dolog.git"
+depends: [
+  "ocaml"
+  "dune"
```

Comment:
```suggestion
  "dune" {>= "1.11"}
```
The `dune-project` file in your repository says this package requires at least dune 1.11